lib/model/app_model/app_utils.dart

```diff
@@ -11,7 +11,7 @@ dynamic witKey(
       if (json[key.snakeCase] == null) {
         return "0";
       }
-      return json[key.snakeCase].split("msat")[0];
+      return json[key.snakeCase].toString();
```

Comment:
Why this choice?

Reply:
because now in the updated version of core lightning the msat value directly comes as integer and we need a string of integer as return value so I added `toString()`  

I mentioned #24

Reply:
we can support both? 

It is not required in this PR but in the future, we can add a method to a string with `.withMsa()` that check if there is a msat string inside and parse it?

or the msat file has a new key? BTW we can discuss in the meeting, put it in your todo list

Reply:
okay sure!!

Reply:
ahhaha yes yes, I want just analyse this case to see if we can can be not depending from version of cln >= 0.12